### ISSUE TITLE
Tricky multi-allele with 2 pathogen species -interspecies complementation

### ISSUE THREAD

**Issue by CuzickA** [PHI-base/curation#30]
For curating Fig 9 in https://github.com/PHI-base/curation/issues/29

I need to make a complicated multi-allele.

_Fusarium graminearum_ has atg9 deleted and heterologous expression of a _Magnaporthe oryzae_ atg9 gene.

The defects of the atg9 deletion in Fg are rescued by the Mo atg9 gene. Interspecies complementation.

Currently I have made a multi-allele for this but could not include both species uniprots (one deleted, one added) so have used the Fg Atg9 Uniprot id for both single alleles and edited the allele name. 

![image](https://user-images.githubusercontent.com/11753634/61646322-1bf0f880-aca1-11e9-8781-89aa64fc82aa.png)

@ValWood do you have a better idea or do we need to wait for the new complementation curation options?

**Comment by ValWood** [comment]
I think we need to wait for complementation for this, it's canonical complementation. Kim is working on this very soon, so if you add a  comment it can be converted,

**Comment by kimrutherford** [comment]
There are some challenges in the pathogen-host case: https://github.com/pombase/canto/issues/1980#issuecomment-512085429

I haven't worked out how to handle that yet.

**Comment by CuzickA** [comment]
Currently attempted in sessions 
https://github.com/PHI-base/curation/issues/33
https://github.com/PHI-base/curation/issues/29

Many of the 'Chemistry resistance' papers contain interspecies complementation and have not been curated. These types of expts have been listed in our FAQ files as expts types that we cannot curate.

Maybe we should delete the draft annotations made in the two sessions above so that we are consistent with the data we are curating and displaying.

What do you think @ValWood?

**Comment by CuzickA** [comment]
Recently in the chemistry curation project we decided that we cannot curate interspecies complementation at this time. We can curate inter strain expts using the allele type 'transformant'. Because of these more recent decisions I am deleting the following older entry as it will be misleading for PHI-base users.

The 'draft' genotype annotations in #33 and #29 have now been recorded within tickets and deleted.

Closing this ticket.
